Paintroid/src/main/java/org/catrobat/paintroid/controller/DefaultToolController.kt

```diff
@@ -119,6 +120,9 @@ class DefaultToolController(
     private fun switchTool(tool: Tool) {
         val currentTool = toolReference.tool
         val currentToolType = currentTool?.toolType
+        if (currentToolType == ToolType.ERASER) {
+            (currentTool as EraserTool).setSavedColor()
```

Comment:
I like the idea! Please also add a testcase for your new implementation.  Cheers

Reply:
> The stroke of the eraser is not visible when erasing. Please take a look at that again. Cheers

Strokes with the erase tool are now visible again during the strokes.

Reply:
I also added some integration tests now